sycl/include/sycl/ext/oneapi/experimental/enqueue_functions.hpp

```diff
@@ -143,9 +143,9 @@ void parallel_for(handler &CGH,
   ext::oneapi::experimental::detail::LaunchConfigAccess<range<Dimensions>,
                                                         Properties>
       ConfigAccess(Config);
-  CGH.parallel_for<KernelName>(ConfigAccess.getRange(),
-                               std::forward<ReductionsT>(Reductions)...,
-                               KernelObj);
+  CGH.parallel_for<KernelName>(
+      ConfigAccess.getRange(), ConfigAccess.getProperties(),
+      std::forward<ReductionsT>(Reductions)..., KernelObj);
```

Comment:
Hi @steffenlarsen , I was wondering if you could share your insight here.

For some context, This PR is a partial implementation of the [sycl_ext_oneapi_cuda_cluster_group](https://github.com/intel/llvm/pull/13594).

This introduces a new runtime launch property, called as `cluster_size`, and if this property is set, it use a new UR entry point, to launch the kernel in a different fashion. I make use of the  functions added in `sycl_ext_oneapi_enqueue_functions` extensions, to pass the properties and launch the kernel.

My question here is, I see `ConfigAccess.getProperties()` was not being passed previously, due to which I suppose any runtime property the user might define, will not get propagated further from what I understand. However, if I do pass these properties, the enqueue functions tests using kernel bundle break on compilation, complaining that `sycl::kernel` is not device copyable ([link](https://github.com/intel/llvm/actions/runs/9602559346/job/26483971253) to failing tests for reference,   I was wondering if I am missing something here, and have an incomplete understanding of the implementation...

Thanks

Reply:
I see, thanks a Lot, 

Well since we do not need the kernel bundle support for now, I can simply remove it in the mean time. 

Thanks